.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "<app name>: <description>"
+labels: kind/feature
+assignees: ''
+
+---
+
+#### Is your feature request related to a problem? Please link issue ticket
+
+Link to existing project ticket that contains a clear and concise description of
+what the problem is. Ex. I'm always frustrated when [...]
+
+#### Describe the solution you'd like
+
+A clear and concise description of what you want to happen.
```

Comment:
I think "Clear and concise is about every other line in this file haha, might want to give example instead